### ISSUE TITLE
[Bug]: When searching and using the placeholder enabled, the table columns breaks

### ISSUE THREAD

**Issue by slossetti** [rappasoft/laravel-livewire-tables#1964]
### What happened?

In my table, when I have the placeholder enabled. `$this->setLoadingPlaceholderEnabled();`

```
class UserTable extends DataTableComponent
{
    protected $model = User::class;

    public function configure(): void
    {
        $this->setPrimaryKey('id');
        $this->setLoadingPlaceholderEnabled();
    }

    public function columns(): array
    {
        return [
            Column::make("Id", "id")
                ->sortable(),
            Column::make("Name", "name")
                ->sortable(),
            Column::make("Email", "email")
                ->sortable(),
            Column::make("Created at", "created_at")
                ->sortable(),
            Column::make("Updated at", "updated_at")
                ->sortable(),
        ];
    }
}
```

and when searching using the search input, the table display breaks when showing results.

Before search
![image](https://github.com/user-attachments/assets/1208bc41-9698-4fa7-b5b0-188b6bc20c4a)

After search
![image](https://github.com/user-attachments/assets/f4c8b792-f52f-4f9e-98da-16167545b1b1)

After remove the searched word
![image](https://github.com/user-attachments/assets/e5a1b632-a4b2-4627-9966-e79c109f724d)

### How to reproduce the bug

`$this->setLoading Placeholder Enabled();`

Then tries to search something.

### Package Version

3.4.20

### PHP Version

8.2.x

### Laravel Version

10 and 11

### Theme

Tailwind 3.x

**Comment by lrljoe** [comment]
Thanks for the bug report, I've identified the issue, and will issue a fix in the next 48 hours.

**Comment by lrljoe** [comment]
This will be fixed in v3.4.22 (to be released shortly)

**Comment by lrljoe** [comment]
Version was released, please reach out if there is still an issue
